WORKSPACE

```diff
@@ -1,25 +1,22 @@
 workspace(name = "angular_material")
 
-# Add NodeJS rules (explicitly used for sass bundle rules)
+# Add Angular source and Bazel rules.
 http_archive(
-  name = "build_bazel_rules_nodejs",
```

Comment:
I've proposed deleting the `rules_nodejs` and `rules_typescript` transitive dependencies here as well (at some point), but since we use them explicitly and not transitively as part of the `ng_module` rule, @jelbourn and I agreed that we should keep them here.

Reply:
Yes- we use these rules directly and should not depend on transitive deps.